server/container_stop.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // StopContainer stops a running container with a grace period (i.e., timeout).
 func (s *Server) StopContainer(ctx context.Context, req *types.StopContainerRequest) error {
-	log.Infof(ctx, "Stopping container: %s", req.ContainerID)
+	log.Infof(ctx, "Stopping container: %s with timeout: %v", req.ContainerID, req.Timeout)
```

Comment:
as timeout is in seconds, I'd add `s`
```suggestion
	log.Infof(ctx, "Stopping container: %s (timeout: %ds)", req.ContainerID, req.Timeout)
```

Reply:
Updated, thanks!